### PR TITLE
#vp 961 export to pdf on linux

### PR DESCRIPTION
**Pull request by kostyrin**

### CODE REVIEW

File: src/VirtoCommerce.OrdersModule.Web/Controllers/Api/OrderModuleController.cs [hunk 1]
```diff
@@ -539,5 +533,28 @@ public async Task<ActionResult<OperationLog[]>> GetOrderChanges(string id)
             return Ok(result);
         }
 
+        private byte[] GeneratePdf(string htmlContent)
```
Comment (tatarincev) [inline src/VirtoCommerce.OrdersModule.Web/Controllers/Api/OrderModuleController.cs]: Can we move all these hardcoded settings into Options?

Comment (kostyrin) [reply src/VirtoCommerce.OrdersModule.Web/Controllers/Api/OrderModuleController.cs]: yes

File: src/VirtoCommerce.OrdersModule.Core/Model/HtmlToPdfOptions.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+namespace VirtoCommerce.OrdersModule.Core.Model
+{
+    public class HtmlToPdfOptions
```
Comment (tatarincev) [inline src/VirtoCommerce.OrdersModule.Core/Model/HtmlToPdfOptions.cs]: How do you think, maybe will be more reasonable to allow change of these settings from the manager?

File: src/VirtoCommerce.OrdersModule.Web/Controllers/Api/OrderModuleController.cs [hunk 3]
```diff
@@ -495,19 +516,25 @@ public async Task<ActionResult> GetInvoicePdf(string orderNumber)
             var message = AbstractTypeFactory<NotificationMessage>.TryCreateInstance($"{notification.Kind}Message");
             message.LanguageCode = order.LanguageCode;
             notification.ToMessage(message, _notificationTemplateRenderer);
+            
+            var uploadPath = Path.GetFullPath(_platformOptions.LocalUploadFolderPath);
```
Comment (tatarincev) [inline src/VirtoCommerce.OrdersModule.Web/Controllers/Api/OrderModuleController.cs]: For resulting pdf I think will be better can use the special folder in the LocalUploadFolderPath

File: src/VirtoCommerce.OrdersModule.Core/Model/HtmlToPdfOptions.cs (hunk 2 above)
Comment (kostyrin) [reply src/VirtoCommerce.OrdersModule.Core/Model/HtmlToPdfOptions.cs]: I'll try to refactor

**Pull request merged**